.eslintrc.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
   parser: '@babel/eslint-parser',
   plugins: ['@metamask/design-tokens'],
   rules: {
-    '@metamask/design-tokens/color-no-hex': 'warn',
+    '@metamask/design-tokens/color-no-hex': 'error',
```

Comment:
Making the no static hex color rule more strict so we are enforcing better brand consistency and design system alignment. If we are using a static color value that is a design smell and should be brought up in #metamask-design-system channel